pkg/controllers/management/drivers/kontainerdriver/kontainerdriver.go

```diff
@@ -450,3 +451,12 @@ func (l *Lifecycle) removeFieldFromCluster(obj *v3.KontainerDriver) error {
 
 	return nil
 }
+
+func RemoveCondition(obj *v3.KontainerDriver, cond condition.Cond) {
+	for i, cond := range obj.Status.Conditions {
+		if cond.Type == cond.Type {
```

Comment:
The local variable `cond` is the same as the function parameter `cond`, so this condition is always `true`.